sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,13 +1,23 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2020-06-10)
```

Comment:
my understanding is that we are releasing on 06/09, no?

Reply:
I wasn't sure of the date so much. 
I am happy to change. Was it TA on 06/09?

Reply:
yes. TA is 06/09

Reply:
I will keep this as 06/10 for now. I will update if we decide otherwise!